mentat/terminal/client.py

```diff
@@ -23,12 +24,14 @@
 class TerminalClient:
     def __init__(
         self,
+        cwd: Path,
```

Comment:
I made `cwd` explicit here since we will be running the "core" library on a separate process for extensions on some unknown filesystem, meaning we can use `os.getcwd` as a default